dotnet/samples/Demos/README.md

```diff
@@ -7,4 +7,5 @@ Demonstration applications that leverage the usage of one or many SK features
 | Create Chat GPT Plugin | A simple plugin that uses OpenAI GPT-3 to chat |
 | Home Automation | This example demonstrates a few dependency injection patterns that can be used with Semantic Kernel. |
 | HuggingFace Image to Text | In this demonstration the application uses Semantic Kernel's HuggingFace ImageToText Service to fetch a descriptive analysis of the clicked image. |
-| Telemetry With Application Insights | Demo on how an application can be configured to send Semantic Kernel telemetry to Application Insights. |
\ No newline at end of file
+| Telemetry With Application Insights | Demo on how an application can be configured to send Semantic Kernel telemetry to Application Insights. |
+| Code Interpreter Plugin | A plugin that leverages Azure Container Apps service to run python code for AI Agents. |
```

Comment:
nit: Agents may be confusing since we have an agents package now.
```suggestion
| Code Interpreter Plugin | A plugin that leverages Azure Container Apps service to execute python code. |
```